doc/reference/tooling/tt_cli/cluster.rst

```diff
@@ -1,10 +1,242 @@
+    *   ``ssl_cert_file`` -- a path to an SSL certificate file.
+    *   ``ssl_ca_file`` -- a path to a trusted certificate authorities (CA) file.
+    *   ``ssl_ca_path`` -- a path to a trusted certificate authorities (CA) directory.
+    *   ``verify_host`` -- verify the certificate’s name against the host. Default ``true``.
+    *   ``verify_peer`` -- verify the peer’s SSL certificate. Default ``true``.
+
+.. _tt-cluster-instance:
+
+Managing configurations of specific instances
+---------------------------------------------
+
+In addition to whole cluster configurations, ``tt cluster`` can manage
+configurations of specific instances within applications. In this case, it operates
+with YAML fragments that describe a single :ref:`instance configuration section <configuration_overview>`.
+For example, the following YAML file can be a source when publishing an instance configuration:
+
+.. code-block:: yaml
+
+    # instance.yaml
+    iproto:
+      listen:
+      - uri: 127.0.0.1:3389
+      threads: 10
+
+To send an instance configuration to a local ``config.yaml``, run ``tt cluster publish``
+with the ``application:instance`` pair as the target argument:
+
+.. code-block:: console
+
+    $ tt cluster publish myapp:instance-002 instance.yaml
+
+To send an instance configuration to a centralized configuration storage, specify
+the instance name in the ``name`` argument of the storage URI:
+
+.. code-block:: console
+
+    $ tt cluster publish "http://localhost:2379/myapp?name=instance-002" instance.yaml
+
+``tt cluster show`` can print configurations of specific cluster instances as well.
+To print an instance configuration from a local ``config.yaml``, use the ``application:instance``
+argument:
+
+.. code-block:: console
+
+    $ tt cluster show myapp:instance-002
+
+To print an instance configuration from a centralized configuration storage, specify
+the instance name in the ``name`` argument of the URI:
+
+.. code-block:: console
+
+    $ tt cluster show "http://localhost:2379/myapp?name=instance-002"
+
+.. _tt-cluster-validation:
+
+Configuration validation
+------------------------
+
+``tt cluster`` can validate configurations against the Tarantool configuration schema.
+
+``tt cluster publish`` automatically performs the validation and aborts in case of an error.
+To skip the validation, add the ``--force`` option:
+
+.. code-block:: console
+
+    $ tt cluster publish myapp source.yaml --force
+
+To validate configurations when printing them with ``tt cluster show``, enable the
+validation by adding the ``--validate`` option:
+
+.. code-block:: console
+
+    $ tt cluster show "http://localhost:2379/myapp" --validate
+
+
+.. _tt-cluster-options:
+
+Options
+-------
+
+..  option:: -u, --username STRING
+
+    A username for connecting to the configuration storage.
+
+..  option:: -p, --password STRING
+
+    A password for connecting to the configuration storage.
+
+.. note::
```

Comment:
Currently it's hard to understand to which option this note relates. Maybe, simply add a short `See also` for both `-u` and `-p`.